src/styles/templates/donation_widget.scss

```diff
@@ -40,7 +39,7 @@ $con-color: #e99;
 	left: 50%;
 	transform: translateX(-50%);
 	background-color: $tab-bg;
-	padding: 0px 8px 3px 8px;
```

Comment:
The donation widget needs to be redone with rems in its entirety. Either remove it for now or finish the job.